### ISSUE TITLE
Revenue Sharing Thoughts

### ISSUE THREAD

**Issue by jbrayton** [appdotnet/api-spec#94]
Regarding Dalton's revenue share proposal at:
http://daltoncaldwell.com/3rd-party-rev-share

I like the spirit of the idea, but I have concerns:
- It seems like this could work very well for mass market clients -- the equivalents of Tweetbot and Twitterrific.  For client apps targeting tiny niche audiences, the revenue from such a revenue sharing system might be inadequate to fund development.  I'm thinking of client apps targeted towards business professionals that might cost, say, $50 to purchase.  The proposal would be fine as long as the developer was free to charge for the client app in addition to, or instead of, participating in the revenue share.
- As a developer, I would want some sort of assurance that the pricing of the service would be stable and consistent.  There might be good reasons to lower the cost of the service to $5/year or to make the service free for light users. That would be fine with me as a user, but I would be unhappy if it diluted the revenue potential from a revenue sharing system.

The idea is intriguing, but I would worry that it would complicate the business relationships in a way that made the system untenable.

John

**Comment by markwilcox** [comment]
I wrote a rather epic length comment on another revenue sharing issue that might be worth reading in this context:
https://github.com/appdotnet/api-spec/issues/44#issuecomment-7753970

I guess for your first point you're referring to this quote from Dalton:

> The advantage is you get to distribute your app for free in the app store, but still get assured you will get financial incentive for building the best software. And, if a user is already an App.net member, they can just sign into your app for free without any money changing hands… the dev still gets paid at the end of the month. I think this creates an incentive for developer to create amazingly high quality apps that people will want to use in the long run, and get away from some of the App Store “chart gaming” going on today.

I'd note that nowhere does anyone say developers HAVE to distribute their apps for free, simply that they can. See the other comment I linked and note that you're also concerned with a somewhat extreme edge case - if you've seen the stats on the number of apps in the App Store above about $10 then you'll realise quite how extreme.  For most apps the demand at a free price point is at least an order of magnitude higher than at $0.99 and anecdotally demand is inversely proportional to price from there. So in almost all cases, even at a $0.20/month revenue share you'll come out ahead giving your app away for free after a short time if the App.net user base is large enough.  Of course at the current size of user base, there's almost no way it'll work out giving your app away for free - you'd need a significant fraction of users to be regularly using your app to cover development costs. Current App.net users are self-selected as willing to pay for quality software and services, so not charging them seems like a wasted opportunity. :)

For your second point I have two responses:
1) As it's a direct revenue share, unless you have additional hosting/support costs, if it makes financial sense for App.net to offer access at a lower price point in terms of price vs total users, that should also make sense for you as a developer.
2) If App.net does offer lower pricing in the future then it should be part of a tiered system. Proposals like #33 and #90 should allow apps to opt-out of being available to users at lower pricing tiers. If part of the lower tier is usage based then certain types of app that naturally have high usage would presumably be excluded from the lower tiers automatically.
